system-parachains/people/people-polkadot/bridge-primitives/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Module with configuration which reflects PeopleHubPolkadot runtime setup
+//! (AccountId, Headers, Hashes...)
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+pub use bp_bridge_hub_cumulus::*;
+use bp_messages::*;
+use bp_runtime::{
+	decl_bridge_finality_runtime_apis, decl_bridge_messages_runtime_apis, Chain, ChainId, Parachain,
+};
+use frame_support::dispatch::DispatchClass;
+use sp_runtime::{RuntimeDebug, StateVersion};
+
+/// PeopleHubPolkadot parachain.
+#[derive(RuntimeDebug)]
+pub struct PeopleHubPolkadot;
```

Comment:
I think we don't use `People Hub` wording, I have never seen it before, so let's better use PeoplePolkadot everywhere
```suggestion
/// PeoplePolkadot parachain.
#[derive(RuntimeDebug)]
pub struct PeoplePolkadot;
```